helpers/GenerateResourcesAndImage.ps1

```diff
@@ -199,7 +231,6 @@ Function GenerateResourcesAndImage {
         -var "client_secret=$($ServicePrincipalClientSecret)" `
         -var "subscription_id=$($SubscriptionId)" `
         -var "tenant_id=$($tenantId)" `
-        -var "object_id=$($spObjectId)" `
```

Comment:
This parameter (`object_id`) seems to be not necessary anymore, as it is calculated automatically from the client / app id.